packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -102,7 +102,8 @@ class _FlutterTestRunnerImpl implements FlutterTestRunner {
         ...<String>['--name', name],
       for (final String plainName in plainNames)
         ...<String>['--plain-name', plainName],
-      '--test-randomize-ordering-seed=$randomSeed',
+      if (randomSeed != null && randomSeed != '0')
+        '--test-randomize-ordering-seed=$randomSeed',
```

Comment:
I think this should just be on null. If someone actually did want random and happened to get 0 this will be confusing